frontend/cypress/unit/search.test.tsx

```diff
@@ -33,43 +32,29 @@ context('Searchbar', () => {
     const spySearch = cy.spy((req: CyHttpMessages.IncomingHttpRequest) => {
       req.reply(keanuSearch.search);
     });
-    const spyGetNodesById = cy.spy(
-      (req: CyHttpMessages.IncomingHttpRequest) => {
-        req.reply(keanuSearch.getNodesById);
-      }
-    );
     cy.intercept(`${apiBaseUrl}/search/all*`, spySearch).as('searchQuery');
-    cy.intercept(`${apiBaseUrl}/getNodesById*`, spyGetNodesById).as(
-      'getNodesQuery'
-    );
 
     // Act 1
     cy.get('.SearchBar').type('keanu');
     // Assert 1
-    cy.wait(['@searchQuery', '@getNodesQuery']).then(() => {
+    cy.wait(['@searchQuery']).then(() => {
```

Comment:
why did you remove that checks?

Reply:
Because this is no longer called any longer. This was the reason to even do these changes.